game/src/test/scala/io/aigar/game/CellSpec.scala

```diff
@@ -58,6 +58,16 @@ class CellSpec extends FlatSpec with Matchers {
     small.acceleration.magnitude should be > big.acceleration.magnitude
   }
 
+  it should "not move slower than the minimal speed no matter its mass" in {
+    val player = new Player(0, Vector2(0f, 0f))
+    val big = new Cell(2, player)
+    player.cells = List(big)
+
+    big.mass = 10000 * Cell.MinMass
+
+    big.maxSpeed should be >= Cell.MinSpeed
```

Comment:
Why not test for equality here? That would test the behaviour that we're really expecting.